components/bl-rating-component/component.json

```diff
@@ -43,32 +51,44 @@
       "defaultValue": "0",
       "handlerId": "defaultValue",
       "dataBinding": true,
-      "handlerLabel": "Default value"
+      "handlerLabel": "Default value logic",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "settingTooltip": "With this handler, you can specify a default rating.",
+      "handlerDescription": "With this handler, you can specify a default rating."
     },
     {
       "name": "iconsAmount",
       "label": "Icons amount",
       "type": "number",
       "defaultValue": "5",
       "handlerId": "iconsAmount",
-      "handlerLabel": "Icons amont",
-      "dataBinding": true
+      "handlerLabel": "Icons amont logic",
+      "dataBinding": true,
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "settingTooltip": "This handler allows you to specify the number of icons to be displayed.",
+      "handlerDescription": "This handler allows you to specify the number of icons to be displayed."
     },
     {
       "type": "color",
       "name": "iconColor",
       "label": "Icon color",
       "handlerId": "iconColorLogic",
       "handlerLabel": "Icon color logic",
-      "defaultValue": "#faaf00"
+      "defaultValue": "#faaf00",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "settingTooltip": "This handler allows you to specify a color for icons.",
+      "handlerDescription": "This handler allows you to specify a color for icons."
     },
     {
       "type": "select",
       "name": "precision",
       "label": "Precision",
       "defaultValue": "1",
       "handlerId": "precision",
-      "handlerLabel": "Precision",
+      "handlerLabel": "Precision logic",
```

Comment:
Pay attention to how '... logic' and '... event' are written at default components.